sdk/iot/provisioning/samples/src/generate_certificate.sh

```diff
@@ -0,0 +1,13 @@
+openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
+openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"
+openssl x509 -text -noout -fingerprint -in device_ec_cert.pem
+
+rm device_cert_store.pem
+cat device_ec_cert.pem >> device_cert_store.pem
+cat device_ec_key.pem >> device_cert_store.pem 
```

Comment:
cat is concatenate files and print on the standard output.

So you can remove these line and replace with following line:

cat device_ec_cert.pem device_ec_key.pem  > device_cert_store.pem